cypress/support/commands.js

```diff
@@ -23,3 +23,15 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (username, password) => {
+  cy.get('[name="username"]')
+    .type(username);
+
+  cy.get('[name="password"]')
+    .type(password);
+});
+
+Cypress.Commands.add('assertMessageShouldExist', (message) => {
+  return cy.get('#flash-messages').contains(message);
```

Comment:
```suggestion
  cy.get('#flash-messages').should('contain', message);
```
we do not need return here because this function returns nothing, just makes some action.
Also, you need to add .should() because it will fail the test if the message  is wrong